tests/queries/0_stateless/02786_parquet_big_integer_compatibility.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+$CLICKHOUSE_LOCAL -q "select toString(424242424242424242424242424242424242424242424242424242::UInt256) as x format Parquet" | $CLICKHOUSE_LOCAL --input-format=Parquet --structure='x UInt256' -q "select * from table"
```

Comment:
It looks suspicious, what `table` here?

Reply:
It's default table name in clickhouse-local

Reply:
But what the data is here? I suppose that request is reading from stdin.

Reply:
Command `clickhouse-local -input-format=... --structure=...` creates table with specified name (`table` is default name) with specified structure and inserts data into it from stdin in specified format. This table can be used in query even in non-interactive mode

Reply:
Thanks for explanation!